packages/browser-client/src/Components/ShowBlockContent.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+  Box,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Td,
+  Th,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionIcon,
+  AccordionPanel,
+} from '@chakra-ui/react'
+import { ethers } from 'ethers'
+import { PortalNetwork } from 'portalnetwork'
+import { useEffect, useState } from 'react'
+
+interface ShowBlockContentProps {
+  portal: PortalNetwork
+}
+
+class Block {
```

Comment:
Lets switch this whole class + `ethers` out for the `@ethereumjs/block` dependency.  We can use the [`BlockHeader.fromRlpSerializedHeader`](https://github.com/ethereumjs/ethereumjs-monorepo/blob/23a7f35c471d7bc3797b3452d9be0947000b6102/packages/block/src/header.ts#L128) static constructor to get all of this data and don't have to duplicate it here.